Get-NetView.psm1

```diff
@@ -334,7 +336,14 @@ function ExecCopyItemsAsync {
         $null = New-Item -ItemType "Container" -Path $Destination
     }
 
-    [String[]] $cmds = $Paths | foreach {"Copy-Item -Path ""$_"" -Destination ""$Destination"" -Recurse -Verbose 4>&1"}
+    [String[]] $cmds = $Paths | foreach {
+        if (Test-Path -Path $_ -PathType Container) {
+            $dirName = Split-Path $_ -Leaf
+            "robocopy `"$_`" `"$Destination\$dirName`" /E /MAX:$(101MB) /V /NP"
```

Comment:
Why robocopy? IIRC, it is a legacy tool whose functionality has already been implemented in standard Windows copy.

Reply:
Doesn't appear to be the case. Maybe you are thinking of xcopy, which is deprecated with robocopy as the replacement.

Reply:
To use robocopy, it must be present in every SKU.  Is that the case here?  IOT, Containers, etc?

Reply:
It is in containers but not IoT Core.